Enterprise-CMCS/macpro-mdct-qmr
# fix local db issues

### Description

It's basically the same PR as [this one](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/pull/11375), but with some extra readme cleanup.

### Related ticket(s)


n/a

---
### How to test

1. Run the app locally (not on cloudfront) and log in as an admin user
2. Create a banner. This should work and be posted/later fetched from the local db.

### Important updates


- Changes dependencies in `/services/database`
- .env files are really unchanged, but i alphabetized them 'cause it's easier to find things that way

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
